docs/vm/backup-restore.md

```diff
@@ -213,3 +213,89 @@ Volumes consume extra disk space in the cluster whenever you create a new virtua
+
+  ![check-schedule-basic.png](/img/v1.4/vm/check-schedule-basic.png)
+
+1. In the **Backups** page, the user can check the status for related VM backups/snapshots that are triggered by the schedule
+
+  ![check-schedule-backups.png](/img/v1.4/vm/check-schedule-backups.png)
+
+1. All the related VM backups/snapshots can be restored as usual
+    - For restoring VM backups, please refer to [VM Backup & Restore](#vm-backup--restore) for more details
+    - For restoring VM snapshots, please see [VM Snapshot & Restore](#vm-snapshot--restore) for more details
+
+  ![check-schedule-restore.png](/img/v1.4/vm/check-schedule-restore.png)
+
+### Edit the Virtual Machine Schedule
+
+1. Go to the **Virtual Machine Schedules** screen
+
+1. Locate the virtual machine schedule, and then select **⋮ > Edit Config**.
+
+  ![edit-schedule-config.png](/img/v1.4/vm/edit-schedule-config.png)
+
+1. User can update **Cron Schedule**, **Retain** and **Max Failure** at runtime.
+
+  ![edit-schedule-parameters.png](/img/v1.4/vm/edit-schedule-parameters.png)
+
+1. Click **Save** to apply the changes
+
+### Suspend/Resume the Virtual Machine Schedule
+
+Users can proactively suspend/resume the schedule during the schedule is active/suspended
+
+1. Go to the **Virtual Machine Schedules** screen
+
+1. Locate the virtual machine schedule, and then select **⋮ > Suspend/Resume**
+    - The schedule could be suspended automatically if the number of `consecutive failed backups/snapshots` reaches **Max Failure**.
+    - For the `backup` type schedule, if the `remote target` is not reachable, the proactively `resume` action will be rejected by the Harvester webhook.
+
+  ![suspend-resume-schedule.png](/img/v1.4/vm/suspend-resume-schedule.png)
+
+### Virtual Machine Backup/Snapshot and Harvester Upgrade
+
+To avoid race conditions between Longhorn volume auto-reattachment logic and make the volume attaching/detaching more resilient, Longhorn introduced a [`volume attachment ticket`](https://github.com/longhorn/longhorn/blob/master/enhancements/20221024-longhorn-volumeattachment.md) implementation. 
+
+By the **volume attachment ticket** design, while a VM backup/snapshot is under processing, it can block the succeeding VM operations (for example, VM live migration). For a VM live migration from a Harvester upgrade, the on-the-fly VM backup can bring the upgrade timeout, and vice versa.
+
+Before applying a Harvester upgrade, the user should make sure there is no VM backup/snapshot under processing and all the VM schedule is suspended, otherwise, the upgrade will be rejected by the Harvester webhook:
+
+  ![upgrade-vmbackup.png](/img/v1.4/vm/upgrade-vmbackup.png)
+
+  ![upgrade-svmbackup.png](/img/v1.4/vm/upgrade-svmbackup.png)
+
+To improve the user experience, Harvester has a future plan to [`automatically suspend/resume all schedules before/after the upgrade`](https://github.com/harvester/harvester/issues/6759).
```

Comment:
This entire section seems to be written for internal stakeholders instead of end users because its focus is implementation design.

@bk201 Which details do we want to expose to end users?

Reply:
I discussed this with @WebberHuang1118  and we can mention the limitation: no backup schedule before an upgrade. There is no need to mention the internal tech details.